Development/nmos/json_fields.h

```diff
@@ -339,6 +339,8 @@ namespace nmos
             const web::json::field_as_array values{ U("values") };
             const web::json::field_as_string validation_fingerprint{ U("validationFingerprint") };
             const web::json::field_as_string status_message{ U("statusMessage") };
+            const web::json::field_as_value data_set{ U("dataSet") }; // NcBulkValuesHolder
+            const web::json::field_as_bool allow_partial{ U("allowPartial") };
```

Comment:
allow_partial ? allow_incomplete ? see https://specs.amwa.tv/nmos-control-feature-sets/branches/publish-device-configuration/device-configuration/#ncbulkpropertiesmanager

Reply:
As above. Allow partial was recently changed to allow incomplete in the specification. Will change.